src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -469,7 +465,7 @@ public void DiagnosticSourceStartStop()
                     Assert.InRange(observer.Activity.Duration, TimeSpan.FromTicks(1), TimeSpan.MaxValue);
 
                     // let's only check that Duration is set in StopActivity, we do not intend to check precision here
-                    Assert.InRange(observer.Activity.StartTimeUtc + observer.Activity.Duration, DateTime.MinValue, DateTime.UtcNow.AddMilliseconds(2 * MaxClockErrorMSec));
+                    Assert.InRange(observer.Activity.Duration, TimeSpan.Zero.Add(TimeSpan.FromTicks(1)), stopWatch.Elapsed.Add(TimeSpan.FromMilliseconds(2 * MaxClockErrorMSec)));
```

Comment:
`TimeSpan.Zero.Add(TimeSpan.FromTicks(1))` is just the same as `TimeSpan.FromTicks(1)`, right?